web-ui/src/pages/AnniversaryReportPage.jsx

```diff
@@ -13,33 +13,47 @@ import SearchBirthdayAnniversaryResults from '../components/search-results/Searc
 import { sortAnniversaries } from '../context/util';
 
 import { selectCsrfToken } from '../context/selectors';
+import { useQueryParameters } from '../helpers/query-parameters';
```

Comment:
The changes here are identical to those in the PR at https://github.com/objectcomputing/check-ins/pull/2301.